pkg/yqutil/yqutil_test.go

```diff
@@ -0,0 +1,28 @@
+package yqutil
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestEvaluateExpression(t *testing.T) {
+	expression := `.cpus = 2 | .memory = "2GiB"`
+	content := `
+# CPUs
+cpus: null
+
+# Memory size
+memory: null
+`
+	// Note: yq currently removes empty lines, but not comments
```

Comment:
Nice, is the empty line issue tracked somewhere in the upstream?

Reply:
* https://github.com/mikefarah/yq/issues/515

Reply:
Could you link that issue in the `// Note:` line?